librz/core/cmd/cmd_print.c

```diff
@@ -4694,6 +4684,46 @@ RZ_IPI RzCmdStatus rz_print_string_c_cpp_handler(RzCore *core, int argc, const c
 	return RZ_CMD_STATUS_OK;
 }
 
+RZ_IPI RzCmdStatus rz_hex_of_assembly_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
+	char *buf = rz_core_hex_of_assembly(core, argv[1]);
+	if (!buf) {
+		return RZ_CMD_STATUS_ERROR;
```

Comment:
please print an error message as well. Here and in the other commands. These are useful to users so they get some feedback if things go wrong.

Reply:
`rz_core_hex_of_assembly` will output the error message when `buf == NULL `. The user can get feedback. The same goes for other commands.